src/raven/spec.clj

```diff
@@ -0,0 +1,9 @@
+(ns raven.spec
+  "specifications for the wire JSON structure when talking to sentry."
+  (:refer-clojure :exclude [assert])
+  (:require [clojure.spec.alpha :as s]
+            [expound.alpha      :as expound]))
+
+; We declare the message spec in the raven.client namespace to allow easy
+; reference from there (simply "::payload" when using the spec).
+(s/def :raven.client/payload (s/keys :req-un [::level ::server_name ::timestamp ::platform]))
```

Comment:
I don't see where `::level`, `::server_name`, `::timestamp`, and `::platform` are defined.

Reply:
As discussed - those just test for key presence (if spec doesn't find a definition for the keyword it assumes we only test for presence of a key in the map).